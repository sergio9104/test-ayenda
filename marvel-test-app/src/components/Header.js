import './Header.scss';

const Header = () => {

    return(
        <header>
            <div className="filterImg">
                <h2>Marvel app</h2>
            </div>
        </header>
    )

}

export default Header;