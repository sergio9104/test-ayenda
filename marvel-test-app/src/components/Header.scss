header{
    position: relative;
    height: 350px;
    width: 100%;
    background-color: red;
    background-image: url(../../public/erik-mclean-8SeJUmfahu0-unsplash.jpg);
    background-size: cover;
    background-position: center;
    background-repeat: no-repeat;
    @media (max-width: 768px) {
        height: 200px;
        width: 100%;
    }
    .filterImg{
        position: absolute;
        height: 100%;
        width: 100%;
        background-color: rgba($color: #000, $alpha: .5);
        display: flex;
        justify-content: flex-start;
        align-items: center;
        h2{
            margin: 0 50px;
            font-size: 100px;
            color: #fff;
            @media (max-width: 425px) {
                font-size: 50px;
            }
        }
    }
}
