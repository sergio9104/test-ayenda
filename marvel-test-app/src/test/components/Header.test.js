import React from 'react'
import { shallow } from "enzyme";
import Header from '../../components/Header';

describe('Pruebas en el comic item component', () => {

    test('Debe mostrar correctamente el componente', () => {
        const wrapper = shallow(<Header />)
        expect( wrapper ).toMatchSnapshot();
    })

})