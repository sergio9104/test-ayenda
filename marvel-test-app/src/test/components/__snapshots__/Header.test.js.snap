// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el comic item component Debe mostrar correctamente el componente 1`] = `
<header>
  <div
    className="filterImg"
  >
    <h2>
      Marvel app
    </h2>
  </div>
</header>
`;
